rules-tests/TypeDeclaration/Rector/ClassMethod/AddReturnTypeDeclarationBasedOnParentClassMethodRector/Fixture/extended_class.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddReturnTypeDeclarationBasedOnParentClassMethodRector\Source\SomeClassWithReturnType;
+
+class MyClass extends SomeClassWithReturnType
+{
+    public function run()
+    {
+    }
+}
+
+?>
+-----
+<?php
+
+use Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddReturnTypeDeclarationBasedOnParentClassMethodRector\Source\SomeClassWithReturnType;
+
+class MyClass extends SomeClassWithReturnType
```

Comment:
I see. The namespace is missing here, should be completed

Reply:
So add to all fixtures

```
namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddReturnTypeDeclarationBasedOnParentClassMethodRector\Fixture;
```

and remove that line from `composer.json`?

Reply:
While the namespace should be added in `Fixture` files, the issue is about resolving `Source` directory, like other registration in composer.json.

But, that's worth a try

Reply:
OK namespaces added and line drom composer.json removed. So we will see.

I also rebased on current master. But there are still some PHPStan errors not related to this PR - it seems they are caused by new rules added to symplify/phpstan-rules.

Reply:
test seems failure https://github.com/rectorphp/rector-src/runs/7468797088?check_suite_focus=true#step:5:84 

please add new commit instead of squash commit so we can see the different between before and after registered in composer.json

Reply:
@samsonasik I noticed it too and it is fixed now. But i squashed commits before I read your message. But it seems that Ci pases now even without change in composer.json.